lib/mix/lib/mix/tasks/format.ex

```diff
@@ -17,6 +17,14 @@ defmodule Mix.Tasks.Format do
   formatter configuration. Evaluating this file should return a keyword list
   with any of the options supported by `Code.format_string!/2`.
 
+  ### Example `.formatter.exs`
+
+  ```
+  [
+    inputs: ["mix.exs", "{config,lib,test}/**/*.{ex,exs}"]
```

Comment:
I think the best default would be:

```elixir
[
  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
]
```

so that the formatter configuration is also formatted :)

Reply:
LGTM! updating